javascript/diff_match_patch_uncompressed.js

```diff
@@ -2407,18 +2418,8 @@ diff_match_patch.patch_obj.prototype.toString = function() {
   return text.join('').replace(/%20/g, ' ');
 };
 
-// CLOSURE:begin_strip
-// Lines below here will not be included in the Closure-compatible library.
-
-// Export these global variables so that they survive Google's JS compiler.
-// In a browser, 'this' will be 'window'.
-// Users of node.js should 'require' the uncompressed version since Google's
-// JS compiler may break the following exports for non-browser environments.
-/** @suppress {globalThis} */
-this['diff_match_patch'] = diff_match_patch;
-/** @suppress {globalThis} */
-this['DIFF_DELETE'] = DIFF_DELETE;
-/** @suppress {globalThis} */
-this['DIFF_INSERT'] = DIFF_INSERT;
-/** @suppress {globalThis} */
-this['DIFF_EQUAL'] = DIFF_EQUAL;
+module.exports = diff_match_patch;
+module.exports['diff_match_patch'] = diff_match_patch;
+module.exports['DIFF_DELETE'] = DIFF_DELETE;
+module.exports['DIFF_INSERT'] = DIFF_INSERT;
+module.exports['DIFF_EQUAL'] = DIFF_EQUAL;
```

Comment:
was this change intentional?

Reply:
This has nothing to do with the fix - it's an older change I made when prototyping a while I started using my fork as a npm package. I will clean this up :)

Reply:
same goes for the package.json

Reply:
I like the change in package.json, but I'm nervous about these lines and if they are okay. did you verify that everything comes through Closure Compiler as it did before?

I guess one simple test is simply to compile with and without this change since no other code is different. If the output compressed bundle is identical then we should be fine.